### PR TITLE
Update swagger.yml

### PR DESCRIPTION
**Pull request by Netmisa**
# Description

This PR update swagger documentation

### CODE REVIEW

**Comment by Netmisa** [comment]
Jenkins could you test this please

File: documentation/swagger.yml [hunk 1]
```diff
@@ -1436,16 +1436,26 @@ definitions:
...
+      name:
         type: "string"
-      priority:
+      max_size:
         type: "integer"
-      wordings:
+      content_type:
+        type: "string"
+      type:
         type: "array"
```
Comment (pthegner) [inline documentation/swagger.yml]: minItems: 1

File: documentation/swagger.yml [hunk 2]
```diff
@@ -1436,16 +1436,26 @@ definitions:
...
+          enum:
+            - "web"
+            - "sms"
+            - "email"
+            - "mobile"
+            - "notification"
+            - "twitter"
+            - "facebook"
+            - "title"
+            - "beacon"
```
Comment (pthegner) [inline documentation/swagger.yml]: Adding required option ?

**Comment by pthegner** [comment]
Next work to do : BOT-441

**Pull request merged**